README.md

```diff
@@ -1,15 +1,55 @@
 <a href="https://codeclimate.com/github/ASAAR/SE2-KaggleComp"><img src="https://codeclimate.com/github/ASAAR/SE2-KaggleComp/badges/gpa.svg" /></a> <a href="https://codeclimate.com/github/ASAAR/SE2-KaggleComp"><img src="https://codeclimate.com/github/ASAAR/SE2-KaggleComp/badges/issue_count.svg" /></a>
 
-# SE2-KaggleComp
+# SE2 Machine Learning Framework
+
 Software Engineering 2 class kaggle competition repo. We'll also be building a bit of a framework to help make doing these easier in the future.
```

Comment:
I say we just get rid of any mention of Software Engineering 2. We know thats what this project is for now, but we intend to grow it out of SE2 at the end of this semester. Might as well start now.